resources/definitions/bfb.def.json

```diff
@@ -0,0 +1,40 @@
+{
+    "id": "bfb",
+    "name": "BFB",
+    "version": 2,
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "Ultimaker",
+        "manufacturer": "BFB",
+        "category": "Other",
+        "file_formats": "text/x-gcode",
+        "platform_offset": [ 0, 0, 0]
+    },
+
+    "overrides": {
+        "speed_topbottom": { "default_value": 40 },
+        "speed_print": { "default_value": 40 },
+        "machine_extruder_count": { "default_value": 1 },
+        "prime_tower_size": { "default_value": 7.745966692414834 },
```

Comment:
DictionaryOfDoom says
```
"prime_tower_size": "math.sqrt(float(wipe_tower_volume) / float(layer_height))",
```
and
```
"wipe_tower_volume":                "15"
```

I have no idea why `wipe_tower_volume` should be 15 by default... I guess that's just what the engine did....